src/infrastructure/persistence/mongodb/MongoDbResponseRepositoryAdapter.ts

```diff
@@ -0,0 +1,40 @@
+import {LoggerPort} from '@core/domain/common/port';
+import Response from '@core/domain/response/Response';
+import { omit } from 'lodash';
+import ResponseRepositoryPort from '../../../domain/response/port/ResponseRepositoryPort';
+import ResponseModel from './ResponseModel';
+
+class MongoDbResponseRepositoryAdapter implements ResponseRepositoryPort {
+  constructor(private readonly logger: LoggerPort) {
+    this.logger = logger;
```

Comment:
You don't need to do this, because of Constructor Assignment from constructor parameters